### PR TITLE
{vis}[foss/2018b]  GraphViz v2.40.1

### PR DESCRIPTION
**Pull request by fizwit**
GraphViz and dependencies.  Requires GObject-Introspection from PR #7039 

Graphviz-2.40.1-foss-2018b.eb
FriBidi-1.0.5-GCCcore-7.3.0.eb
GTS-0.7.6-GCCcore-7.3.0.eb
Pango-1.42.4-foss-2018b.eb

### CODE REVIEW

File: easybuild/easyconfigs/g/Graphviz/Graphviz-2.40.1-foss-2018b.eb [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+description = """Graphviz is open source graph visualization software. Graph
+ visualization is a way of representing structural information as diagrams of
+ abstract graphs and networks. It has important applications in networking,
+ bioinformatics,  software engineering, database and web design, machine
+ learning, and in visual interfaces for other technical domains."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://graphviz.gitlab.io/pub/graphviz/stable/SOURCES/']
+sources = ['graphviz.tar.gz']
```
Comment (boegel) [inline easybuild/easyconfigs/g/Graphviz/Graphviz-2.40.1-foss-2018b.eb]: This is just... stupid. At the very least, we should rename this to include a version, for example via:

```python
sources = [{'download_filename': 'graphviz.tar.gz', 'filename': SOURCELOWER_TAR_GZ}]
```

@fizwit Can you refer me to the open issue?

File: easybuild/easyconfigs/g/GTS/GTS-0.7.6-GCCcore-7.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+ functions to deal with 3D surfaces meshed with interconnected triangles."""
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['059c3e13e3e3b796d775ec9f96abdce8f2b3b5144df8514eda0cc12e13e8b81e']
+
+builddependencies = [
+    ('pkg-config', '0.29.2'),
```
Comment (boegel) [inline easybuild/easyconfigs/g/GTS/GTS-0.7.6-GCCcore-7.3.0.eb]: @fizwit `binutils` *must* be included as a build dependency whenever `GCCcore` is used as a toolchain, so please include `('binutils', '2.30')` here as well

Comment (fizwit) [reply easybuild/easyconfigs/g/GTS/GTS-0.7.6-GCCcore-7.3.0.eb]: ('binutils', '2.30') added to GTS

File: easybuild/easyconfigs/g/Graphviz/Graphviz-2.40.1-foss-2018b.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/g/Graphviz/Graphviz-2.40.1-foss-2018b.eb]: @fizwit Ping on the issue that discusses the lack of properly versioned source tarballs?

Comment (boegel) [reply easybuild/easyconfigs/g/Graphviz/Graphviz-2.40.1-foss-2018b.eb]: see https://gitlab.com/graphviz/graphviz/issues/1371

**Pull request closed without merging**